build/conformance.textproto

```diff
@@ -329,6 +329,19 @@ requirement: {
   whitelist_regexp: "test/test/util/util.js"
 }
 
+# Disallow the general use of AbortController, which is not available on all
+# supported platforms.
+requirement: {
+  type: BANNED_NAME_CALL
+  value: "AbortController"
+  value: "window.AbortController"
+  error_message:
+    "Using \"AbortController\" directly is not allowed; "
+    "use PendingRequest.abort instead."
+  whitelist_regexp: "lib/net/http_fetch_plugin.js"
+  whitelist_regexp: "test/test/util/util.js"
```

Comment:
That test util doesn't reference the AbortController type. Was this perhaps a copy-paste error?

Reply:
Exactly. Fixed now!